bin/vscout.dart

```diff
@@ -1,54 +1,73 @@
+                     /@@@@@@@                /@@@@@@@.,@@@@@@@@@@@@@@@@@@@@@@@@/                   
+                      /@@@@@@@              /@@@@@@&..@@@@@@@*        /@@@@@@@(                    
+                       .@@@@@@@,           %@@@@@@& .@@@@@@@/        *@@@@@@@.                     
+                         @@@@@@@/        ,@@@@@@@* %@@@@@@@         (@@@@@@%                       
+                          %@@@@@@*      .@@@@@@@* #@@@@@@@         &@@@@@@&                        
+                           %@@@@@@%    .@@@@@@@/ %@@@@@@@.        &@@@@@@&                         
+                            /@@@@@@#  /@@@@@@@.   /@@@@@@@.      %@@@@@@(                          
+                             ,@@@@@@@%@@@@@@@      ,@@@@@@@,      %@@@@*                           
+                              ,@@@@@@@@@@@@% /.      @@@@@@@#      %@@                             
+                                @@@@@@@@@@% *@@/      @@@@@@@(      *.                             
+                                 @@@@@@@@/ %@@@@*     .&@@@@@@(                                    
+                                  (@@@@@,.&@@@@@@*      #@@@@@@@                                   
+                                   #@@@,.@@@@@@@/        *@@@@@@@,                                 
+                                    ,@ .@@@@@@@/        ,@@@@@@@*                                  
+                                      *@@@@@@@.        ,@@@@@@@*                                   
+                                      .&@@@@@@*       %@@@@@@@                                     
+                                        &@@@@@@%     %@@@@@@@                                      
+                                         @@@@@@@%   #@@@@@@#                                       
+                                          #@@@@@@# @@@@@@@%                                        
+                                           /@@@@@@@@@@@@@(                                         
+                                            .@@@@@@@@@@@.                                          
+                                             .@@@@@@@@@,                                           
+                                              .&@@@@@@                                             
+                                                %@@@&                                              
+                                                 %@/                                               
+                                                  .                                                                                                                                                                                          
+  """);
+  print('Welcome to Vscout');
+  print('Robotics scouting software');
+  print('For more information, visit\nhttps://vscout.readthedocs.io');
```

Comment:
Why 3 different print statements?

Reply:
https://dart.dev/guides/language/effective-dart/usage#do-use-adjacent-strings-to-concatenate-string-literals